src/app/comments/comment/comment.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-comment',
+  templateUrl: './comment.component.html',
+  styleUrls: ['./comment.component.scss']
+})
+export class CommentComponent {
+  @Input() comment: any;
```

Comment:
Maybe strong type to a string?

Reply:
It's not a string, it's an object. Technically, we should strong-type it into a class, but since we are likely going to change what the comment object contains, I didn't really strong type it. 

I do recognize that the any is pretty useless as it is.